Dockerfile

```diff
@@ -123,7 +123,7 @@ ADD . ${APP_ROOT}
 
 ## Setup environment
 RUN ${APPLIANCE_ROOT}/setup && \
-    echo "export PATH=\$PATH:/opt/rubies/ruby-2.3.1/bin" >> /etc/default/evm && \
+    echo "export PATH=\$PATH:/opt/rubies/ruby-2.4.2/bin" >> /etc/default/evm && \
```

Comment:
Can you extract the Docker changes into a separate PR (same reasons as I stated in https://github.com/ManageIQ/manageiq-appliance-build/pull/229#issuecomment-333892541 ).  This way we can merge that separately from the Travis/Gemfile bump.